bundles/Billing/Resources/assets/pages/HomePage.js

```diff
@@ -189,7 +189,7 @@ class HomePage extends Component {
                 keyEvent: 'posted',
                 items: invoices
                     .filter((item) => {
-                        return item.recorded === true && item.exportedDate === null;
+                        return item.recorded === true;
```

Comment:
This should be `.filter(item => item.recorded === true)` or just `.filter(item => item.recorded)` (if we know that `item.recorded` is a boolean value).